data/distro.yml

```diff
@@ -794,3 +794,30 @@
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>
       </li>
     </ol>
+    
+    
```

Comment:
```suggestion
```

Also, please remove the four spaces on previous line to an empty line without any spaces.